### ISSUE TITLE
README's build instructions do not work without additional dependencies met

### ISSUE THREAD

**Issue by transitorykris** [fastly/cli#35]
`staticcheck`, `golint`, and `gosec` are required dependencies. Ideally, the documentation should reflect that so the user can install them, or have the build process take care of it.

```
% make
gofmt -l ./cmd ./pkg
go vet ./{cmd,pkg}/...
go: downloading github.com/blang/semver v3.5.1+incompatible
go: downloading github.com/fastly/go-fastly v1.7.0
go: downloading gopkg.in/src-d/go-git.v4 v4.13.1
go: downloading github.com/BurntSushi/toml v0.3.1
go: downloading github.com/google/go-cmp v0.3.1
go: downloading github.com/segmentio/textio v1.2.0
go: downloading github.com/mitchellh/go-wordwrap v1.0.0
go: downloading gopkg.in/alecthomas/kingpin.v3-unstable v3.0.0-20180810215634-df19058c872c
go: downloading github.com/kennygrant/sanitize v1.2.4
go: downloading github.com/mholt/archiver v3.1.1+incompatible
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/mholt/archiver/v3 v3.3.0
go: downloading github.com/dustinkirkland/golang-petname v0.0.0-20191129215211-8e5a1ed0cff0
go: downloading golang.org/x/crypto v0.0.0-20191029031824-8986dd9e96cf
go: downloading github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
go: downloading golang.org/x/net v0.0.0-20200226121028-0de0cce0169b
go: downloading github.com/mattn/go-colorable v0.1.4
go: downloading github.com/nicksnyder/go-i18n v1.10.1
go: downloading github.com/klauspost/pgzip v1.2.1
go: downloading github.com/nwaples/rardecode v1.0.0
go: downloading github.com/andybalholm/brotli v0.0.0-20190621154722-5f990b63d2d6
go: downloading github.com/ajg/form v1.5.1
go: downloading github.com/google/go-github/v28 v28.1.1
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/dsnet/compress v0.0.1
go: downloading github.com/mattn/go-isatty v0.0.10
go: downloading github.com/ulikunitz/xz v0.5.6
go: downloading gopkg.in/src-d/go-billy.v4 v4.3.2
go: downloading github.com/sergi/go-diff v1.0.0
go: downloading github.com/google/jsonapi v0.0.0-20200226002910-c8283f632fb7
go: downloading golang.org/x/sys v0.0.0-20191120155948-bd437916bb0e
go: downloading github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8
go: downloading github.com/google/go-querystring v1.0.0
go: downloading github.com/pelletier/go-toml v1.2.0
go: downloading github.com/klauspost/compress v1.9.2
go: downloading gopkg.in/yaml.v2 v2.2.8
go: downloading github.com/pierrec/lz4 v2.3.0+incompatible
go: downloading github.com/golang/snappy v0.0.1
go: downloading github.com/golang/gddo v0.0.0-20190419222130-af0f2af80721
go: downloading github.com/src-d/gcfg v1.4.0
go: downloading github.com/emirpasic/gods v1.12.0
go: downloading github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading gopkg.in/warnings.v0 v0.1.2
go: downloading github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
go: downloading github.com/xanzy/ssh-agent v0.2.1
staticcheck ./{cmd,pkg}/...
/bin/bash: staticcheck: command not found
make: *** [staticcheck] Error 127
```

**Comment by phamann** [comment]
Thanks for the raising the issue @transitorykris, this has been fixed as a side-effect of #26, as we have now changed the default Make target to only build the binary and not run `all`. Hope that helps.

**Comment by phamann** [comment]
This has now been fixed and released in #26.
